nav2_costmap_2d/include/nav2_costmap_2d/costmap_filters/binary_filter.hpp

```diff
@@ -118,6 +125,26 @@ class BinaryFilter : public CostmapFilter
 
   bool default_state_;  // Default Binary Filter state
   bool binary_state_;  // Current Binary Filter state
+
+  // List of binary params to be changed
+  std::vector<std::string> binary_parameters_;
+
+  // Parameter information
+  struct BinaryParameter
+  {
+    std::string node_name;    // Node to which the parameter belongs
+    std::string param_name;   // Name of parameter
+    bool default_state;       // Parameter default state
+  };
+
+  // List of params with info
+  std::vector<BinaryParameter> binary_parameters_info_;
+  // List of clients for changing parameters
+  std::vector<rclcpp::Client<rcl_interfaces::srv::SetParameters>::SharedPtr>
+  change_parameters_clients_;
```

Comment:
This will cause linting falures - please correct the indents:
```
  std::vector<rclcpp::Client<rcl_interfaces::srv::SetParameters>::SharedPtr>
    change_parameters_clients_;
```

Reply:
To be honest I get the opposite result:
```
5: Code style divergence in file 'include/nav2_costmap_2d/costmap_filters/binary_filter.hpp':
5: 
5: --- include/nav2_costmap_2d/costmap_filters/binary_filter.hpp
5: +++ include/nav2_costmap_2d/costmap_filters/binary_filter.hpp.uncrustify
5: @@ -140,2 +140,2 @@
5: -  std::vector<rclcpp::Client<rcl_interfaces::srv::SetParameters>::SharedPtr> 
5: -    change_parameters_clients_;
5: +  std::vector<rclcpp::Client<rcl_interfaces::srv::SetParameters>::SharedPtr>
5: +  change_parameters_clients_;
5: 
5: 1 files with code style divergence
```

What am I missing?

Reply:
I think he was intuiting. You have a couple more linting failures https://app.circleci.com/pipelines/github/ros-planning/navigation2/10484/workflows/affb9099-bea7-40c7-b616-6c4316ee29c1/jobs/33103/tests then good to go

Reply:
Yes, that was some kind of intuitive suggestion for better readability. Anyway, since we need to follow linting guidelines, this could be remained w/o modification